### DISCUSSION TITLE
Почему Zapret отправляет страйки на видео? Это автор или кто-то действует от его имени?

### DISCUSSION THREAD

**Discussion by skallekss** [bol-van/zapret#737] [General]
Здравствуйте, у меня возник вопрос по поводу страйков, которые были выставлены на YouTube в связи с использованием открытого исходного кода из репозитория Zapret. Я создал обучающее видео, используя материал из этого репозитория, так как он находился в открытом доступе на GitHub, и не преследовал коммерческих целей. Однако, неожиданно получил страйк на YouTube за демонстрацию кода.

Хотелось бы понять, действительно ли сам автор, или кто-то от его имени, отправляет такие страйки? Или, возможно, это недоразумение?

![image](https://github.com/user-attachments/assets/2bbed03e-0d3e-4362-a486-7479f65581c9)

**Comment by Guru1-228** [comment]
Интересно чья это по факту почта

**Comment by bol-van** [comment]
Это не автор, а самозванец. Код zapret полностью свободен и имеет MIT лицензию.
https://github.com/bol-van/zapret/issues/736

**Reply by skallekss** [reply]
![image](https://github.com/user-attachments/assets/6d4db1d9-435d-4a2b-8285-342b52986a16)
После почты просил в телегу написать ему, вот что из этого вышло, скорее всего стиллер

**Reply by bol-van** [reply]
Не скорее всего, а точно. Прав так поступать ему никто не давал.
Он мог как угодно распространять zapret, но он не имел права что-то запрещать другим.
А так же не указан оригинальный копирайт

> 
> The above copyright notice and this permission notice shall be included in all
> copies or substantial portions of the Software.
>

**Reply by NewUse** [reply]
наверное можно попросить GitHub и Google забанить и почтовый ящик и акк на Ютюбе.

**Reply by skallekss** [reply]
Не забанит его гугл, гуглу похуй на это, он тебе будет слать копи пасту то что не вмешивается в конфликты 3-х лиц. Это bol-van'у по факту писать надо и говорить что он владелец репозитория, а не ноунейм какой-то, но этим он заниматься не будет, поэтому вариант один - соглашаться, а потом просто сносить видео. И НИКАК ТЫ гуглу не докажешь. что болван настоящий владелец, а не тот клоун, ему сайт чат гпт писал xd)

**Comment by bol-van** [comment]
Предостережение о скаммерах добавлено в основной readme большими буквами прямо в самое начало

**Reply by MaksimNikolashin** [reply]
Уважаемый разработчик программы @bol-van. У авторов ютуб каналов, которые показывали как работает программа и помогали людям разблокировать ютуб в России появились проблемы из-за мошенников, которые кидают страйки и угрожают удалением каналов. Они заставляют снимать по 2 видео на свои каналы для того чтобы в случае чего могли отправить еще по 2 страйка и тогда за 3 страйка гугл удалит канал. Авторы под давлением могу поддаться на этот шантаж и выполнить условия мошенников дабы спасти канал. Но это усугубит ситуация. 
После того как мы писали встречную апелляцию в гугл в отдел copyrightyotube от гугл стали приходить отказы во встречной жалобе и просьбе снять страйки и ограничения на каналы. Хотя в письме мы указывали все необходимые пруфы и скриншоты переписки с мошенниками. 
Могу ли я вас попросить от лица всех блогеров и авторов каналов, которые пострадали от мошенников написать письмо в гугл и предоставить доказательство что вы настоящий автор этой программы и что по лицензии блогеры могут снимать контент про вашу программу. Это очень поможет нам и я думаю мы всегда пойдем на встречу вам, если вам будет нужна помощь блоггеров в виде взаимовыгодного сотрудничества. Спасибо автор канала "Максим Николашин"

**Reply by bol-van** [reply]
У меня нет никаких доказательств авторства программы, кроме этого репозитория.
Идти и писать свою фамилию и номер телефона не буду ни при каких обстоятельствах.

**Reply by numbereleven-a** [reply]
> Уважаемый разработчик программы @bol-van. У авторов ютуб каналов, которые показывали как работает программа и помогали людям разблокировать ютуб в России появились проблемы из-за мошенников, которые кидают страйки и угрожают удалением каналов. Они заставляют снимать по 2 видео на свои каналы для того чтобы в случае чего могли отправить еще по 2 страйка и тогда за 3 страйка гугл удалит канал. Авторы под давлением могу поддаться на этот шантаж и выполнить условия мошенников дабы спасти канал. Но это усугубит ситуация. После того как мы писали встречную апелляцию в гугл в отдел copyrightyotube от гугл стали приходить отказы во встречной жалобе и просьбе снять страйки и ограничения на каналы. Хотя в письме мы указывали все необходимые пруфы и скриншоты переписки с мошенниками. Могу ли я вас попросить от лица всех блогеров и авторов каналов, которые пострадали от мошенников написать письмо в гугл и предоставить доказательство что вы настоящий автор этой программы и что по лицензии блогеры могут снимать контент про вашу программу. Это очень поможет нам и я думаю мы всегда пойдем на встречу вам, если вам будет нужна помощь блоггеров в виде взаимовыгодного сотрудничества. Спасибо автор канала "Максим Николашин"

а на лицензию ютубу пофиг? https://github.com/bol-van/zapret/blob/master/docs/LICENSE.txt

**Reply by IndeecFOX** [reply]
@MaksimNikolashin чтож вы на своём канале распространяете ссылку на сайт gitrok под своими видео. Вас шантажируют? Вы решили что ваш небольшой канал важнее людей, которые могут потерять данные на их устройствах и денежные средства?

**Comment by MaksimNikolashin** [comment]
Да. Пишут что недостаточно информации.

**Reply by bol-van** [reply]
А у страйкеров ее достаточно ? У них есть более весомые доказательства авторства ?

**Reply by bol-van** [reply]
пусть посмотрят кто делает коммиты тут все время. кто пишет код.
и если он тут оставил файл лицензии , значит вряд ли хотел что-то запрещать

**Reply by numbereleven-a** [reply]
> пусть посмотрят кто делает коммиты тут все время. кто пишет код. и если он тут оставил файл лицензии , значит вряд ли хотел что-то запрещать

думаю у них есть нормативы для обжалования страйка и без них они ничего делать не будут и в эти нормативы входят то что они просят, а если их нет то информации недостаточно, свободный запад

**Reply by bol-van** [reply]
Так почему они не применяли эти же нормативы к самим страйкерам ?
У них так заведено, что пожаловаться может любой, и мы лучше профилактически прирежем, чтобы потом нас не засудили ?
То есть презумция виновности по умолчанию ?
Даже по нашим законам при обращении в мос гор суд по копирайтному-блокировочному вопросу нужно предоставить доказательства владения правами

**Reply by skallekss** [reply]
Потому-что ютуб - контора пидорасов, 2-ю неделю им написываю, ответ один и тот же "Мы получили от вас встречное уведомление, но не можем его рассмотреть, так как вы предоставили недостаточно информации."

**Comment by remittor** [comment]
@beer-code-ss 
> Однако, неожиданно получил страйк на YouTube за демонстрацию кода.

Так покажите нам свой заблоченный видос. Врятли за показ кода с гитхаба гугл одобрил страйк (хотя у них автоматическая система почти всегда их одобряет).
Просто по вашему видосу можно примерно понять, какую причину для страйка придумал злоумышленник.

Похожий случай был описан в ТГ: R1XONE_CHANNEL (посты про скамера `@zapretgit` уже удалены, звуковухи остались).
"У меня в видосе есть даже ссылка на **автора этой хуйни**": https://t.me/R1XONE_CHAT/61143

Но посты про РКН остались:
Даже РКН просит удалить обучающий видос: https://t.me/R1XONE_CHAT/60908 + https://t.me/R1XONE_CHAT/60917

<details><summary>требования РКН</summary>

![image](https://github.com/user-attachments/assets/f9047500-49d0-493e-a095-17a9d3fe27c4)

![image](https://github.com/user-attachments/assets/a6e91e6d-a276-4e22-82f7-0ab4cdfb5f07)

</details>

**Comment by IndeecFOX** [comment]
Собаки (мягко выражусь) распространяют через ТГ канал с десятками тысяч подписчиков t.me/peekcsgo а не только на youtube. Это первоисточник. При комментариях сразу банят.
Счётчик на сайте походу реальный, так что минимум 40 тысяч компов вполне могут быть под угрозой. Спасибо РКН что защитил людей. 10/10. Когда-то это всё схлопнется и застиллят много всего, а пока радостные юзеры обсуждают и радуются ;(
Кинул абузу хостеру и регистратору домена. Мб прокатит хотя бы это.
Так же решил использовать РКН по прямому назначению. Коль он банит полезные способы блокировок, то почему бы не забанить доступ к явно вредоносному. Написал обращение: https://eais.rkn.gov.ru/feedback/

Ответ хостера:
> Здравствуйте.
> Ваша жалоба получена и передана нашему клиенту.
> Если у Вас остались какие-либо вопросы, пожалуйста, пишите на [abuse@pindc.ru](mailto:abuse@pindc.ru).
> С уважением, служба мониторинга жалоб!
